GlobalStaticVersion.props

```diff
@@ -8,12 +8,12 @@
     <SemanticVersionMajor>2</SemanticVersionMajor>
     <SemanticVersionMinor>2</SemanticVersionMinor>
     <SemanticVersionPatch>0</SemanticVersionPatch>
-    <PreReleaseMilestone>beta6</PreReleaseMilestone>
+    <PreReleaseMilestone></PreReleaseMilestone>
     <!-- 
       Date when Semantic Version was changed. 
       Update for every public release.
     -->
-    <SemanticVersionDate>2016-06-02</SemanticVersionDate>
+    <SemanticVersionDate>2016-12-05</SemanticVersionDate>
```

Comment:
Okay, so we want it to still output something like 2.2.0.850, got it.  I always like file versions with .0 on the end but I see how this would conflict with updating from a beta.